### DISCUSSION TITLE
[Feature] View last added photos

### DISCUSSION THREAD

**Discussion by AFumi39** [immich-app/immich#12305] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I miss a feature I used a lot, first in Google Photo (years ago) and then in Synology Photo (until last week), and strangely I couldn't find in Immich. It's the option to view the last added (or uploaded) images.
There's no way to sort the timeline other than photo date, and there's no alternative (like a "dummy" album that show last 100 images by add/upload date). But if I select (just for example) a new album to upload in the mobile app, it could contain both new and old photos, and I have no chance to know which are (so I can't manage them properly). Or if I edit an old photo, the DateTimeOriginal is the same as before, and it will be put somewhere in the middle of my timeline.

I tried to search in the forum and on reddit (also asked a question there), but with no luck, so I think that the feature is really missing and nobody already asked for it

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8716

**Reply by AFumi39** [reply]
Oh... I didn't managed to find it using the search tool... 🤦‍♂️
Sorry for duplicated request
